config/settings.yml.example

```diff
@@ -44,8 +44,8 @@
 #:daemon_pid: /var/run/foreman-proxy/foreman-proxy.pid
 
 # host and ports configuration
-# host to bind ports to (possible values: *, localhost, 0.0.0.0)
-#:bind_host: '*'
+# comma-separated list of interfaces to bind ports to (possible values: *, localhost, 0.0.0.0)
```

Comment:
Nitpick, it might be safest to say "array" rather than "comma-separated" so it isn't misinterpreted as `bind_host: eth1,eth` or something (i.e. comma-separated string).